src/subsplit_dag.hpp

```diff
@@ -129,13 +129,14 @@ class SubsplitDAG {
   // Get Taxon's bitset clade positional id.
   size_t GetTaxonId(const std::string &name) const;
   // Get node based on node id.
-  SubsplitDAGNode *GetDAGNode(const size_t node_id) const;
+  SubsplitDAGNode GetDAGNode(const size_t node_id) const;
+  MutableSubsplitDAGNode GetDAGNode(const size_t node_id);
```

Comment:
For my clarification:
So it will use the always use the mutable `GetDAGNode()` if SubsplitDAG is not `const`, is that correct?

Reply:
Correct. I chose to keep the name SubsplitDAGNode instead of ConstSubsplitDAGNode to avoid changing too many locations. In the future I'd like to go trough the codebase and ensure const-correctness of all SubsplitDAGNode usage locations.